strawberry_django/filters.py

```diff
@@ -72,6 +74,19 @@ def filter_deprecated(model, *, name=None, lookups=False):
     return filter(model, name=name, lookups=lookups)
 
 
+@lru_cache
+def get_field_names_from_opts(opts):
+    '''
+    copy from django.db.models.sql.query.get_field_names_from_opts
+    because get_field_names_from_opts not in django.db.models.sql.query.__all__
+    and no cache optimization
+    '''
+    return set(chain.from_iterable(
+        (f.name, f.attname) if f.concrete else (f.name,)
+        for f in opts.get_fields()
+    ))
```

Comment:
It's not a problem that the function is not in `__all__`. That only matters when you do `from <some_place> import *`

Just import it from there and use it directly :)

Reply:
I think functions not in __all__ are meant for internal use only and may be renamed or removed later